scripts/post-commit

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+SCRIPT_PATH="$(dirname "$0")/$(dirname "$(readlink "$0")")"
+
+# Import utils
+source ${SCRIPT_PATH}/utils/message
+
+title "Lint commit message..."
+
+linting_errors="$(npm run conventional-changelog-lint --silent -- -e; echo x$?)"
+
+if [  ${linting_errors##*x} -ne 0 ]
+then
+  warning "Wrong commit message format"
+  echo -e "Please adjust the commit message following the formatting rules" \
+    "as documented in our contributing guidelines.\n${linting_errors%x*}"
```

Comment:
We can even include the link https://github.com/dcos/dcos-ui/blob/master/CONTRIBUTING.md